docker-compose.yaml

```diff
@@ -0,0 +1,35 @@
+version: '3'
+
+services:
+  mongodb:
+    image: mongo
+    container_name: mongodb
```

Comment:
I will rename the containers on the master branch such that anyone with multiple containers running will not get confused as multiple projects might contain a mongo database